roles/ceph-defaults/tasks/facts.yml

```diff
@@ -221,15 +221,20 @@
 - block:
     - name: get current cluster status (if already running)
       command: "{{ docker_exec_cmd }} ceph --cluster {{ cluster }} -s -f json"
+      changed_when: false
+      failed_when: false
+      check_mode: no
```

Comment:
Actually, this command should be executed on a monitor node:

```
  run_once: true
  delegate_to: "{{ groups[mon_group_name][0] }}"
  when:
    - not rolling_update
    - groups.get(mon_group_name, []) | length > 0
```

Reply:
done